testsuite/azure-arc-conformance.properties

```diff
@@ -0,0 +1,45 @@
+# Azure arc conformance test properties file
+# Please maintain the structure of the file
+# Do not add inline comments
+
+# configure Azure storage where result will be uploaded
+# name of your storage account (will be provided)
+az-storage-account=
```

Comment:
Should we add basic param validation for the mandatory params in the 'k8s-conformance-test-suite.sh' script? It would be better if we can give proper error message to partners if any such param is missing

Reply:
ack